.flake8

```diff
@@ -89,6 +89,11 @@ per-file-ignores =
   # have <= members in tests (WPS202), including many local vars (WPS210),
   # additionally test docstrings don't need param lists (DAR, DCO020):
   tests/**.py: DAR, DCO020, S101, S404, S603, WPS202, WPS210, WPS436, WPS442, WPS450
+  # (B950) is ignored due to it erroring on help text which is displayed in the API.
+  # While they can be shorter, they are not a priority to shorten.
```

Comment:
```suggestion
```

This rule specifically already allows going 10% over the line length limit in a few places. But not in most of a module's contents.

I think I mentioned this elsewhere already — I expect that newly added code would not increase the number of linting violations.

I've fixed this in some (most?) places already. This is not because of the help text, this is because of the heavy indentation chosen. If you move the braces on separate lines, I'm pretty sure `autopep8` will reformat it for you, similar to https://github.com/ansible/awx-plugins/blob/82c8c95/src/awx_plugins/credentials/dsv.py#L8-L82. I noticed that it suggests highly indented formatting if you don't do this.
Hinting `autopep8` what the formatting preference are like that, can help you save some manual formatting efforts if it goes into wrong direction.

In general, long lines suggest that code might be getting hard to consume, so I recommend avoiding ignoring such rules at all cost.

Reply:
change committed per previous review.

Reply:
Looks like it was never actually pushed?